sdks/hermes-engine/hermes-engine.podspec

```diff
@@ -12,7 +12,7 @@ import_hermesc_file=File.join(__dir__, "..", "hermesc", "osx-bin", "ImportHermes
 # package.json
 package_file = File.join(__dir__, "..", "..", "package.json")
 package = JSON.parse(File.read(package_file))
-version = package['version']
+version = '0.70.0'
```

Comment:
Should this be 70.1?

Reply:
Nope - we're using the Hermesc build artifact from RN 0.70.0, but our package version is 0.70.1